pkg/trace/api/debugger.go

```diff
@@ -23,34 +23,48 @@ const (
 	// logsIntakeURLTemplate specifies the template for obtaining the intake URL along with the site.
 	logsIntakeURLTemplate = "https://http-intake.logs.%s/api/v2/logs"
 
-	// logsIntakeMaximumTagsLength is the maximum number of characters we send as ddtags.
-	logsIntakeMaximumTagsLength = 4001
+	debuggerDiagnosticsURLTemplate = "https://debugger-intake.%s/api/v2/debugger"
+
+	// intakeMaximumTagsLength is the maximum number of characters we send as ddtags.
+	intakeMaximumTagsLength = 4001
```

Comment:
I suggest `intakeMaximumTagLength` (singular) instead of `intakeMaximumTagsLength` since otherwise it sounds like the number of tags, instead of the length of a single tag.

Reply:
It's actually the length of the string value we set on the `ddtags` query parameter 😃 

I've changed it to `intakeDDTagsQueryStringMaximumLength` and updated the doc, what do you think?

Reply:
would just `ddTagsMaxLen` be sufficiently precise? (with a good godoc to go alongside it)
That's just a very wordy variable name for Go :)

Reply:
I settled on `ddTagsQueryStringMaxLen` - I think having the query string part is important since it doesn't affect the `ddtags` inside the event json 🙂